docs/pages/kubernetes-access/helm/guides/custom.mdx

```diff
@@ -0,0 +1,273 @@
+migrating your setup from an older version of the Helm charts.
+
+### Install Helm
+
+Teleport's charts require the use of Helm version 3. You can [install Helm 3 by following these instructions](https://helm.sh/docs/intro/install/).
+
+Throughout this guide, we will assume that you have the `helm` and `kubectl` binaries available in your `PATH`:
+
+```bash
+$ helm version
+version.BuildInfo{Version:"v3.4.2"}
+
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"17+"}
+Server Version: version.Info{Major:"1", Minor:"17+"}
+```
+
+### Add the Teleport Helm chart repository
+
+To allow Helm to install charts which are hosted in the Teleport Helm repository, use `helm repo add`:
+
+```bash
+helm repo add teleport https://charts.releases.teleport.dev
+```
+
+Once installed, you can search repositories for charts using the `helm search repo` command:
+
+```bash
+helm search repo teleport
+```
+
+To update the cache of charts from the remote repository, run `helm repo update`:
+
+```bash
+helm repo update
+```
+
+### Setting up a Teleport cluster with Helm using a custom config
+
+In `custom` mode, the `teleport-cluster` Helm chart does not create a `ConfigMap` containing a `teleport.yaml` file for you, but
+expects that you will provide this yourself.
+
+For this example, we'll be using this `teleport.yaml` configuration file (with appropriately complex static tokens):
+
+```bash
+cat << EOF > teleport.yaml
+teleport:
+  log:
+    output: stderr
+    severity: INFO
+
+auth_service:
+  enabled: true
+  cluster_name: custom.example.com
+  tokens:
+  - "proxy,node:$(tr -dc A-Za-z0-9 </dev/urandom | head -c 32)"
+  - "trusted_cluster:$(tr -dc A-Za-z0-9 </dev/urandom | head -c 32)"
+  listen_addr: 0.0.0.0:3025
+  public_addr: custom.example.com:3025
+
+proxy_service:
+  enabled: true
+  listen_addr: 0.0.0.0:3080
+  public_addr: custom.example.com:443
+
+ssh_service:
+  enabled: true
+  labels:
+    cluster: custom
+  commands:
+  - name: kernel
+    command: [/bin/uname, -r]
+    period: 5m
+EOF
+```
+
+<Admonition type="tip">
+  You can skip this step if you already have a `teleport.yaml` file locally that you'd like to use.
+</Admonition>
+
+You can create the namespace for the config and add the `teleport.yaml` from your local disk like this:
+
+```bash
+kubectl create namespace teleport
+kubectl --namespace teleport create configmap teleport --from-file=teleport.yaml
+```
+
+<Admonition type="note">
+  The name of the `ConfigMap` used must match the name of the Helm release that you install below (the name just after `helm install`).
+  In this example, it's `teleport`.
+
+  The name (key) of the configuration file uploaded to your `ConfigMap` must be `teleport.yaml`. If your configuration file is named differently
+  on disk, you can specify the key that should be used in the `kubectl` command:
+
+  ```bash
+  $ kubectl --namespace teleport create configmap teleport --from-file=teleport.yaml=my-teleport-config-file.yaml
```

Comment:
Nit: sometimes we use `$` and sometimes not, probably remove elsewhere too?

Reply:
Yeah you're right. My intention is to use `$` whenever there is example output after the command (to separate the two) but this one was inconsistent.